CONTRIBUTING.md

```diff
@@ -118,7 +118,7 @@ source $HOME/.profile
 cd determined
 python3 -m venv $HOME/.virtualenvs/determined
 . $HOME/.virtualenvs/determined/bin/activate
-$HOME/.virtualenvs/determined/bin/python3.7 -m pip install --upgrade pip
+$HOME/.virtualenvs/determined/bin/python3.9 -m pip install --upgrade pip
```

Comment:
is 3.9 our current min?

Reply:
It's 3.7 (as listed). The previous instructions have users installing python 3.9 ... whichever version is used I think it should be consistent.